helm/app/values.yaml

```diff
@@ -129,3 +131,13 @@ pgBouncer:
   limits:
     cpu: 100m
     memory: 32Mi
+
+certbot:
+  image:
+    tag: 1.0.0
+    pullPolicy: IfNotPresent
+  certbot:
+    email: '' # The value must be passed in via the deploy script
+    server: '' # The value must be passed in via the deploy script
+  cron:
+    suspend: true # until certs are issued
```

Comment:
are you going to manually run the job?

Reply:
Yes, intending to manually trigger the cronjob and PR to remove this once all 3 certs are issued